lib/bulk_upload/bulk_upload.rb

```diff
@@ -3,12 +3,24 @@
 module BulkUpload
   module BulkUpload
     def self.add_location_data(data, organisation)
-      data.each do |row|
-        location = organisation.locations.new(address: row[0], postcode: row[1])
-        row.slice(2..).each do |ip_address|
+      data_without_header = data.drop(1)
+
+      data_without_header.each do |row|
+        location = organisation.locations.new(address: build_address(row), postcode: row[5]&.strip)
+        row.slice(6..).reject(&:blank?).each do |ip_address|
           location.ips.new(address: ip_address)
         end
       end
     end
+
+    def self.build_address(row)
+      addres_1 = row[0]
```

Comment:
there's a lot of magic numbers here that correspond to upload_form.rb - line 10. Could we do something about that?

Reply:
@koetsier Im not sure what you mean? Can you please give me an example. Thanks

Reply:
The fact that the first address line is in column 0 and the postcode in column 5 is encoded in https://github.com/alphagov/govwifi-admin/blob/de7bd0180f06b541ac78d6d21c21c1c3ced81765/app/models/upload_form.rb#L10 - I think it would be a good idea to have one source of truth of what the structure of the file looks like and then refer to that instead of to column numbers. That was it is easier to change and harder to make mistakes